src/sentry/static/sentry/app/components/stream/groupRowActions.tsx

```diff
@@ -0,0 +1,204 @@
+          </ActionWrapper>
+
+          <ActionWrapper>
+            <Tooltip title={t('Ignore')}>
+              <IgnoreActions
+                onUpdate={this.handleUpdate}
+                shouldConfirm={false}
+                inboxRowAction
+              />
+            </Tooltip>
+          </ActionWrapper>
+
+          <ActionWrapper>
+            <StyledDropdownLink
+              key="actions"
+              caret={false}
+              className="btn btn-sm btn-default action-more"
+              customTitle={<IconEllipsis size="xs" />}
+              title=""
+              anchorRight
+            >
+              <MenuItem noAnchor>
+                <Projects orgId={orgId} slugs={[group.project.slug]}>
+                  {({projects, initiallyLoaded, fetchError}) => {
+                    const project = projects[0];
+                    return (
+                      <ResolveActions
+                        hasRelease={
+                          project.hasOwnProperty('features')
+                            ? (project as Project).features.includes('releases')
+                            : false
+                        }
+                        latestRelease={
+                          project.hasOwnProperty('latestRelease')
+                            ? ((project as Project).latestRelease as Release)
+                            : undefined
+                        }
+                        orgId={orgId}
+                        projectId={group.project.id}
+                        onUpdate={this.handleUpdate}
+                        shouldConfirm={false}
+                        inboxRowAction
+                        disabled={!!fetchError}
+                        disableDropdown={!initiallyLoaded || !!fetchError}
+                        projectFetchError={!!fetchError}
+                      />
+                    );
+                  }}
+                </Projects>
+              </MenuItem>
+              <MenuItem divider />
+              <MenuItem noAnchor>
+                <ActionLink
+                  className="action-remove-bookmark"
+                  onAction={() => this.handleUpdate({isBookmarked: false})}
+                  shouldConfirm={false}
+                  title={t('Remove from Bookmarks')}
+                >
+                  {t('Remove from Bookmarks')}
+                </ActionLink>
+              </MenuItem>
+              <MenuItem divider />
+              <MenuItem noAnchor>
+                <ActionLink
+                  className="action-delete"
+                  onAction={this.handleDelete}
+                  shouldConfirm={false}
+                  title={t('Delete')}
+                >
+                  {t('Delete')}
+                </ActionLink>
+              </MenuItem>
+            </StyledDropdownLink>
+          </ActionWrapper>
+        </HoverActionsWrapper>
+      </Wrapper>
+    );
+  }
+}
+
+const Wrapper = styled('div')`
+  position: absolute;
```

Comment:
shouldn't need this anymore, same with background and stuff

Reply:
👍